dll/win32/devmgr/lang/ro-RO.rc

```diff
@@ -201,7 +205,7 @@ BEGIN
             LVS_NOSORTHEADER | WS_BORDER | WS_TABSTOP, 7, 46, 209, 80
     LTEXT "Furnizor:", -1, 14, 134, 50, 8
     EDITTEXT IDC_FILEPROVIDER, 66, 134, 155, 8, NOT WS_TABSTOP | NOT WS_BORDER | ES_AUTOHSCROLL | ES_READONLY
-    LTEXT "Versiune fișier:", -1, 14, 150, 50, 8
+    LTEXT "Versiunea fișierului:", -1, 14, 150, 50, 8
```

Comment:
There is a problem with this one:

![image](https://user-images.githubusercontent.com/578406/190930680-a567d5ca-f6c9-4382-97fc-301159647b51.png)

Reply:
It seems that the text I modified got to be too long for being displayed in the dialog box, so I reverted my change. Instead of "Versiunea fișierului" is "Versiune fișier" again.

Reply:
If the new version of text is better, you can just resize the elements.

Reply:
Also the last line about driver signature is cut off. It would really make sense if you fix that while you're at it.

Reply:
And how can I resize the elements?

Reply:
> And how can I resize the elements?

You take the generated binary, open it in ay resource editor (e.g. ResHacker), find the resource corresponding to the dialog in romanian, try redimensioning (mostly in width) the necessary elements so that text can fit in them, and once you've found the improved dimensions, you adjust those in the resource text file in your PR so that when you recompile the binary, the dialog(s) show correctly.

Reply:
Finally, I was able to modify the resource corresponding to the dialog in Romanian language. Thank you everyone for the help!

Reply:
One of dialogs has been fixed, but not the one I have requested changes for. :sweat_smile: Alright I will probably fix this one myself.

Reply:
Ok, thank you!